test/checks/keyboard/no-focusable-content.js

```diff
@@ -70,4 +75,33 @@ describe('no-focusable-content tests', function() {
     assert.deepEqual(checkContext._data, { messageKey: 'notHidden' });
     assert.deepEqual(checkContext._relatedNodes, [params[2].children[0]]);
   });
+
+  it('should return true if element has content which is natively focusable and has a widget role but is disabled', function() {
+    var vNode = queryFixture(
+      '<button id="target"><a href="foo.html" disabled>Hello</a></button>'
```

Comment:
Hmm, that shouldn't have worked. Links don't have a disabled attribute. I opened a separate issue for that (#3315). No need to fix that in this PR. But we should change the example. I suggest:

```suggestion
      '<button id="target"><input value="hello" disabled></button>'
```